pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -588,7 +588,18 @@ class HassTypeHintChecker(BaseChecker):  # type: ignore[misc]
             "Used when method return type is incorrect",
         ),
     }
-    options = ()
+    options = (
+        (
+            "ignore-missing-annotations",
+            {
+                "default": True,
+                "type": "yn",
+                "metavar": "<y or n>",
```

Comment:
It could make sense to invert the flag, i.e. to be `False` by default. With that `--<flag>` would be enough. I couldn't come up with a good name though, `check-code-without-annotations` might be a bit long.

In the end, it's only for testing anyway, so shouldn't matter much.

Reply:
Naming is always hard!

Would "include-unannotated" be better ?

Reply:
If we can find a proper good name, then I'm inclined to just merge.

I'm not sure how many will actually use it. I use it after every plugin tweak to improve the general code base but I doubt anyone else does...

Reply:
> If we can't find a proper good name, then I'm inclined to just merge.
> 
> I'm not sure how many will actually use it. I use it after every plugin tweak to improve the general code base but I doubt anyone else does...

My thinking as well. If it works for you, I think the current name / option is good enough.